docs/operate/validators/validator-signatures-aws.mdx

```diff
@@ -14,12 +14,12 @@ If you're only intending to run a Validator for testing or development purposes,
 
 Follow the instructions in [Agent Keys](../set-up-agent-keys.mdx) to generate an AWS IAM user and either a KMS or a hexadecimal key. You will use this key's credentials in the following steps.
 
+### 2. Create an S3 bucket
+
 :::note
```

Comment:
just to make it more obvious
```suggestion
:::info
```
<img width="511" alt="image" src="https://github.com/user-attachments/assets/6a627bc4-75f2-4113-9879-94f47e32dd0a">